data/records/lhcb-ntuples-hackathon.json

```diff
@@ -0,0 +1,1022 @@
+[
+  {
+    "abstract": {
+      "description": "<p>Data from proton-proton (pp) collisions collected by the LHCb experiment filtered to produce Ntuples for {insert reason for request}.</p><p>FIXME Here should come a long explanation of inputs, ntuples, etc.</p><p>FIXME If we can attach the <code>info.yaml</code> etc, this is the place wher they would be added."
+    },
+    "accelerator": "CERN-LHC",
+    "collaboration": {
+      "name": "LHCb collaboration"
+    },
+    "collections": ["LHCb-Collision-Datasets"],
+    "date_created": ["2010"],
+    "date_published": "2025",
+    "date_reprocessed": "2025",
+    "experiment": ["LHCb"],
+    "publisher": "CERN Open Data portal",
+    "recid": "93900",
+    "relations": [
+      {
+        "recid": "93901",
+        "type": "isParentOf"
+      },
+      {
+        "recid": "93902",
+        "type": "isParentOf"
+      },
+      {
+        "recid": "93903",
+        "type": "isParentOf"
+      },
+      {
+        "recid": "93904",
+        "type": "isParentOf"
+      },
+      {
+        "recid": "93905",
+        "type": "isParentOf"
+      },
+      {
+        "recid": "93906",
+        "type": "isParentOf"
+      },
+      {
+        "recid": "93907",
+        "type": "isParentOf"
+      },
+      {
+        "recid": "93908",
+        "type": "isParentOf"
+      }
+    ],
+    "title": "LHCb Ntuples from user request 142",
+    "type": {
+      "primary": "Dataset",
+      "secondary": ["Collision"]
+    },
+    "usage": {
+      "description": "<p>You can clone and amend this ntupling request here:",
```

Comment:
I suggested a modification to encourage folks to check out the existing Ntuples and only clone and amend the request if these Ntuples do not suit their needs.